processor/error/event.go

```diff
@@ -94,7 +94,7 @@ func (e *Event) addException() {
 		e.enhancer.Add(ex, "code", e.Exception.Code.(string))
 	}
 
-	st := e.Exception.Stacktrace.Transform()
+	st, _ := e.Exception.Stacktrace.Transform()
```

Comment:
The information should also be handled for `errors`.

Reply:
here we're ignoring the boolean for only_library_frames since we don't use that on error traces - should we be?

Reply:
@roncohen, @sqren does it make sense to also provide the `only_library_frames` information for errors?